package.json

```diff
@@ -33,6 +33,7 @@
   "scripts": {
     "test": "node test",
     "bench": "node test --bench",
-    "build": "uglifyjs lib/marked.js -cm  --comments /Copyright/ -o marked.min.js"
+    "build": "uglifyjs lib/marked.js -cm  --comments /Copyright/ -o marked.min.js",
+    "preversion": "npm run build"
```

Comment:
I think it's best committing the minified file too

Reply:
This should be covered by the `npm version` updating the package.json. Both files need to be committed at the same time.